ide-support/revsblibrary.livecodescript

```diff
@@ -702,6 +702,98 @@ function revSBTargetToPlatform pTarget
    end if
 end revSBTargetToPlatform
 
+-- item 1 of the architectures should be the default architecture
+constant kWindowsArchitectures = "x86-32"
+constant kMacOSXArchitectures = "x86-32,x86-64"
+constant kLinuxArchitectures =  "x86-32,x86-64,armv6-hf"
+constant kAndroidArchitectures =  "armv6"
+constant kiOSArchitectures =  "armv7,arm64"
+constant kEmscriptenArchitectures =  "javascript"
```

Comment:
OK, I'll change. This architecture stuff isn't actually used for mobile or html5 but in the event it is one day it would be good if it were correct!